commander/src/bootstrapping/commands/start.ts

```diff
@@ -170,10 +170,17 @@ export abstract class StartCommand extends Command {
 		config.version = this.config.pjson.version;
 		// Inject other properties specified
 		if (flags['api-ipc']) {
-			config.rpc = { enable: flags['api-ipc'], mode: 'ipc' };
+			config.rpc = utils.objects.mergeDeep({}, config.rpc, {
+				enable: flags['api-ipc'],
+				mode: 'ipc',
+			});
 		}
 		if (flags['api-ws']) {
-			config.rpc = { enable: flags['api-ws'], mode: 'ws', port: flags['api-ws-port'] };
+			config.rpc = utils.objects.mergeDeep({}, config.rpc, {
+				enable: flags['api-ws'],
+				mode: 'ws',
+				port: flags['api-ws-port'],
```

Comment:
What if user only provided `--api-ws` flag and not `--api-ws-port`? I am not sure what would be behaviour of the config if `port` is specified as undefined. So may may avoid adding port to config unless if specified.

Reply:
In that case, `flags['api-ws-port']` will be undefined and use the default value?

Reply:
I tested `lisk-utils` behaviour for `mergeDeep` using `undefined` and `null` values and it works.